python/GafferImageUI/ImageReaderUI.py

```diff
@@ -191,6 +199,21 @@
 
 		],
 
+		"colorSpace" : [
+
+			"description",
+			"""
+			The colour space of the input image.
```

Comment:
Could we clarify here that this will be used to convert to linear space during loading?